src/main/java/com/minecolonies/coremod/colony/jobs/JobShepherd.java

```diff
@@ -0,0 +1,84 @@
+package com.minecolonies.coremod.colony.jobs;
+
+import com.minecolonies.coremod.client.render.RenderBipedCitizen;
+import com.minecolonies.coremod.colony.CitizenData;
+import com.minecolonies.coremod.entity.ai.basic.AbstractAISkeleton;
+import com.minecolonies.coremod.entity.ai.citizen.shepherd.EntityAIWorkShepherd;
+import net.minecraft.entity.passive.EntitySheep;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.List;
+
+/**
+ * Created by Asher on 3/9/17.
+ */
+public class JobShepherd  extends AbstractJob
+{
+
+    /**
+     * The water the fisherman is currently fishing at
+     * Contains the location of the water so that the fisherman can path to the fishing spot.
+     */
+    private List<EntitySheep> sheep;
+
+    /**
+     * Instantiates the placeholder job.
+     * @param entity the entity.
+     */
+    public JobShepherd(final CitizenData entity)
+    {
+        super(entity);
+    }
+
+    @NotNull
+    @Override
+    public String getName()
+    {
+        return "com.minecolonies.coremod.job.Shepherd";
+    }
+
+    /**
+     * Generate your AI class to register.
+     *
+     * @return your personal AI instance.
+     */
+    @Nullable
+    @Override
+    public AbstractAISkeleton<JobShepherd> generateAI()
+    {
+        return new EntityAIWorkShepherd(this);
+    }
+
+    /**
+     * Get the RenderBipedCitizen.Model to use when the Citizen performs this job role.
+     *
+     * @return Model of the citizen.
+     */
+    @NotNull
+    @Override
+    public RenderBipedCitizen.Model getModel()
+    {
+        return RenderBipedCitizen.Model.FARMER;
+    }
+
+    /**
+     * Getter for current sheep.
+     *
+     * @return Location of the current sheep.
+     */
+    public List<EntitySheep> getSheep()
+    {
+        return sheep;
+    }
+
+    /**
+     * Setter for current sheep.
+     *
+     * @param sheep New location for the current sheep.
+     */
+    public void setSheep(final List<EntitySheep> sheep)
+    {
+        this.sheep = sheep;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "sheep". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)